lighthouse-core/lib/emulation.js

```diff
@@ -113,9 +120,19 @@ function goOffline(driver) {
   return driver.sendCommand('Network.emulateNetworkConditions', OFFLINE_METRICS);
 }
 
+function enableCPUThrottling(driver) {
+  return driver.sendCommand('Emulation.setCPUThrottlingRate', CPU_THROTTLE_METRICS);
+}
+
+function disableCPUThrottling(driver) {
+  return driver.sendCommand('Emulation.setCPUThrottlingRate', NO_CPU_THROTTLE_METRICS);
+}
+
```

Comment:
FWIW an early patch for cpu throttling allowed the user to put in their octane results. We would use those to compare against saved results for mobile devices.